windows/internal/smoke_test.bat

```diff
@@ -3,7 +3,7 @@ set SRC_DIR=%~dp0
 pushd %SRC_DIR%\..
 
 if "%CUDA_VERSION%" == "102" call internal\driver_update.bat
-if "%CUDA_VERSION%" == "110" call internal\driver_update.bat
+@REM if "%CUDA_VERSION%" == "110" call internal\driver_update.bat
```

Comment:
Why do we comment out this line?

Reply:
You are right, it's not needed. I was thinking about something wrong.

Reply:
I reverted the change